src/rocm_docs/rocm_docs_theme/flavors/rocm/left-side-menu.jinja

```diff
@@ -1,3 +1,3 @@
 {%
-set main_doc_link = ("ROCm API & tools list", projects['rocm'] ~ "/reference/library-index.html")
+set main_doc_link = ("ROCm docs homepage", projects['rocm'] ~ "index.html")
```

Comment:
This is intended to point to the list, either API or tools

eg:

projects['rocm'] ~ "reference/api.html"

Reply:
in the latest ROCm docs iteration, I split the APIs and the tools, so thought it best to just link back to the ROCm docs homepage